src/styles/_frames.scss

```
@@ -0,0 +1,9 @@
+@keyframes moveRound {
+  0% {
+    transform: rotate(180deg);
+  }
+
+  100% {
+    transform: rotate(540deg);
```

```suggestion
    transform: rotate(180deg);
  }

  100% {
    transform: rotate(540deg);
```
Why not 0 and 360 deg?

---

becouse they starts position is on 6:00, and i made 180 - 540 to start on 00:00